video-presentation.py

```diff
@@ -230,6 +233,43 @@ def parsevalue(size, max_size):
         self.logger.debug("Masks after preparing: %s", parsed_masks)
         return parsed_masks
 
+    def create_video_previews(self, filename):
+        """Create mp4 and webm heavily compressed previews of the presentation to use in the previewer"""
+
+        # Let's not be greedy, use half available cores since we are probably in a docker container
+        encoding_threads = multiprocessing.cpu_count()
```

Comment:
does this give the correct cpu count if your inside a cpuset cgroup?

Reply:
I'm pretty certain the answer is no. By default `ffmpeg` will take everything it can see and I'm not sure it is cgroup aware either. Probably the right thing to do is actually limit the CPU count available to the container in the docker-compose file.

Reply:
Well, what docker does it using a cgroup cpuset no?

Reply:
Should I just drop this and let docker handle it?